website/docs/reference/sinks/logdna.md.erb

```diff
@@ -0,0 +1,21 @@
+<%- component = metadata.sinks.logdna -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
+
+<%= options(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list) %>
+
+<%- end -%>
+## How It Works [[sort]]
```

Comment:
Anything special we should call out here?

Reply:
Don't think so? I would say this sink follows the very basic http batching sink design. The only special stuff is with some of the config options but that should be documented already with the options description.